### PR TITLE
feat: content migration prep

### LINKED ISSUE #764: Remove white backgrounds from icons in resource cards
### Browser

Chrome, Safari, Firefox

### Description

https://next.carbondesignsystem.com/assets/guidelines/accessibility/overview#tools

Thre are white backgrounds/canvases behind icon artwork that appear on hover

current
Screen Shot 2022-05-20 at 5 11 25 PM
Screen Shot 2022-05-20 at 5 11 18 PM
Screen Shot 2022-05-20 at 5 11 29 PM

this one is also too small
Screen Shot 2022-05-20 at 5 11 35 PM

expected (https://carbondesignsystem.com/guidelines/accessibility/overview/#tools)
Screen Shot 2022-05-20 at 5 11 58 PM
Screen Shot 2022-05-20 at 5 12 03 PM

### Steps to reproduce

go [here](https://next.carbondesignsystem.com/assets/guidelines/accessibility/overview#tools ) and hover over resource cards

### PR DESCRIPTION
**Pull request by alisonjoseph**
Ref #751
Closes #764

This splits out some of the updates not dependent on remote mdx from https://github.com/carbon-design-system/carbon-platform/pull/768

#### Changelog

**Nav/pages**

- Add About section to nav
- Move roadmap from landing page to /about-carbon/platform-roadmap
- Add Designing section to nav 
- Rename Guidelines -> Elements **(this is why 300+ files were changed)**
- Remove all L0

**MDX / fixes**
- Add Zenhub to icon package
-  Import commonly used carbon components to be available for use on mdx pages
- Various mdx component style fixes

#### Testing / reviewing

New roadmap page http://localhost:3000/about-carbon/platform-roadmap
Landing page localhost:3000/
Elements pages http://localhost:3000/elements/2x-grid/overview
MDX http://localhost:3000/mdx

### CODE REVIEW

File: services/web-app/pages/assets/index/index.module.scss [hunk 1]
```diff
@@ -236,6 +236,15 @@
...
 
+.intro-copy {
+  margin-top: spacing.$spacing-11;
+  @include type.type-style('fluid-heading-03', true);
+
+  &:last-child {
+    margin-bottom: spacing.$spacing-10;
+  }
+}
+
```
Comment (francinelucca) [inline services/web-app/pages/assets/index/index.module.scss:247]: is this being user? can't find it 😅

File: services/web-app/pages/designing/get-started.mdx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: Get started
+description:
+---
```
Comment (francinelucca) [inline services/web-app/pages/designing/get-started.mdx:4]: is this supp to be empty? should we add a "coming soon" or something ?

**Review by francinelucca — COMMENTED** [review]
mostly LGTM, some comments

File: services/web-app/pages/designing/get-started.mdx (hunk 2 above)
Comment (alisonjoseph) [reply services/web-app/pages/designing/get-started.mdx:4]: I can add add coming soon, this is what Andrea is working on here https://github.com/carbon-design-system/carbon-platform/pull/784

File: services/web-app/pages/assets/index/index.module.scss (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/pages/assets/index/index.module.scss:247]: Yea the roadmap page, but now that you point that out it probably makes sense to pull the styles from here into their own file.

Comment (alisonjoseph) [reply services/web-app/pages/assets/index/index.module.scss]: @francinelucca updated

**Review by francinelucca — DISMISSED** [review]
👍 👍

**Review by andreancardona — CHANGES_REQUESTED** [review]
super small comment

**Comment by alisonjoseph** [comment]
@andreancardona good catch!! Fixed 😄

**Review by andreancardona — APPROVED** [review]
🔥

**Review by francinelucca — APPROVED** [review]

**Pull request merged**